independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -2803,6 +2813,126 @@
         ]
       }
     },
+    "/crm/v3/objects/contacts/search": {
+      "post": {
+        "responses": {
+          "default": {

```

Comment:
@ryanedshannon it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
Thank you for the review and feedback. I've updated with a set of responses. Let me know if this is sufficient.